server/src/routes/swagger.ts

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+/**
+ * This file is used as a base for the swagger complition in the index.js file of routes.
```

Comment:
I would write this as:
`This file is used as a base for the Swagger compilation in routes/index.ts.`